cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/TranslationConfiguration.kt

```diff
@@ -93,7 +93,9 @@ private constructor(
      * always take priority over those in the whitelist.
      */
     val includeBlocklist: List<Path>,
-    passes: List<Pass>,
+    passes: List<KClass<out Pass<*>>>,
+    val replacedPasses:
```

Comment:
Some documentation could be useful. What if there are different replacements of a pass for different languages? If I see it correctly, this would require a `Map<KClass<out Pass<*>>, Map<KClass<out Language<*>>, KClass<out Pass<*>>>>`

Reply:
Should be addressed now, I didn't completely understand it before :) Now it is possible to have different replacements of a pass for different languages. As suggest by you, I settled on a map with the pair of (oldPass, language) as key and the replacement as value - should be most efficient for looking it up.